Assets/Scripts/OrchestraArmy/Entity/Entities/Players/WeaponSelection/Weapon/Data/WeaponWheelPlaceholderData.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using OrchestraArmy.Entity.Entities.Players.WeaponSelection.Weapon.Weapons.Factory;
+using UnityEngine;
+
+namespace OrchestraArmy.Entity.Entities.Players.WeaponSelection.Weapon.Data
+{
+    [Serializable]
+    public class WeaponWheelPlaceholderData
+    {
+        
+        [SerializeField]
+        private bool _unlocked = false;
+        
+        [SerializeField]
+        private WeaponType _weaponType = WeaponType.Guitar;
+
+        private IWeapon _weapon;
+        
+        /// <summary>
+        /// Boolean indicating whether the weapon is unlocked or not. 
+        /// </summary>
+        public bool Unlocked { 
+            get => _unlocked;
+            set => _unlocked = value;
+        }
+        
+        /// <summary>
+        /// The type of the weapon which this placeholder holds.
+        /// </summary>
+        public WeaponType WeaponType { get; set; }
```

Comment:
Hier staat een WeaponType en boven staat een _weaponType. Vgm is dat niet de bedoeling? iig sinds ze niets met elkaar te maken hebben. Laat WeaponType gebruik maken van _weaponType!

Reply:
Lijkt me niet de bedoeling nee, maar was jouw code ;)

Zal t ff aanpassen

Reply:
ssssh hoeft niemand te weten ;)